tests/demo/pytest/test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+source /sdf/group/lcls/ds/ana/sw/conda1/manage/bin/psconda.sh
+
+cd /sdf/home/h/hzhang12/demo/pytest
+
+python test_*.py
```

Comment:
Use pytest, not python here. pytest will provide more and better info on the process